src/gsIO/gsOptionList.cpp

```diff
@@ -52,6 +52,54 @@ bool gsOptionList::getSwitch(const std::string & label) const
     return it->second.first;
 }
 
+std::vector<std::string> gsOptionList::getMultiString(const std::string & gn) const
+{
+    GISMO_ASSERT(hasGroup(gn), "Invalid request (getGroup): The group " + gn + " does not exist.");
+
+    std::vector<std::string> result;
+
+    const std::string search = gn + ".";
+
+    // add strings to vector
+    for (StringTable::const_iterator it = m_strings.begin(); it != m_strings.end(); it++)
+        if (util::starts_with(it->first, search) && !util::ends_with(it->first, "Size"))
+            result.push_back(it->second.first);
+
```

Comment:
This does not guarantee the order to be correct.

It might be better to do something like:

int sz = getInt(search + "Size");
std::vector<std::string> result;
result.reserve(sz);
for (int i=0; i<sz; ++i)
    result.push_back(getString(search + util::to_string(i)));
return result;